src/main/java/uk/gov/hmcts/reform/sscs/services/ccd/CcdCasesSender.java

```diff
@@ -105,6 +105,23 @@ public boolean updateLanguage(Long caseId, IdamTokens idamTokens, String languag
         }
     }
 
+    public boolean updateProcessingVenue(Long caseId, IdamTokens idamTokens, String venue) {
+        try {
+            var startEventResponse = ccdClient.startEvent(idamTokens, caseId, MIGRATE_CASE);
+            var caseData = sscsCcdConvertService.getCaseData(startEventResponse.getCaseDetails().getData());
+            caseData.setProcessingVenue(venue);
```

Comment:
this line is the only difference between this method and the interpreter one right? we could probably pass it as a lambda  expression to the update method